pkgs/os-specific/linux/kernel-headers/default.nix

```diff
@@ -90,12 +92,24 @@ let
         # `$(..)` expanded by make alone
         "HOSTCC:=$(CC_FOR_BUILD)"
         "HOSTCXX:=$(CXX_FOR_BUILD)"
+      ]
+      ++ lib.optionals (stdenvNoCC.buildPlatform.is32bit || true) [
+        # To properly detect LFS flags 32-bit build environments like
+        # pkgsi686Linux.linuxHeaders Kbuild uses this Makefile bit:
+        #     HOST_LFS_CFLAGS := $(shell getconf LFS_CFLAGS 2>/dev/null)
+        #
+        # `getconf` is not available in early bootstrap and thus the
+        # build fails on filesystems with 64-bit inodes as:
+        #     linux-headers> fixdep: error fstat'ing file: scripts/basic/.fixdep.d: Value too large for defined data type
+        #
+        # Let's hardcode the output of `getconf` for this case.
+        "HOST_LFS_CFLAGS=-D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64"
```

Comment:
I think just `-D_FILE_OFFSET_BITS=64` should be enough?

Reply:
Yeah, `-D_FILE_OFFSET_BITS=64` is enough. I dropped bits related to `structuredAttrs` as it does not need whitespace handling any more.